bindings/ocaml/Hacl.ml

```diff
@@ -18,9 +18,8 @@ module Hacl_HMAC = Hacl_HMAC_bindings.Bindings(Hacl_HMAC_stubs)
 module Hacl_Poly1305_32 = Hacl_Poly1305_32_bindings.Bindings(Hacl_Poly1305_32_stubs)
 module Hacl_HKDF = Hacl_HKDF_bindings.Bindings(Hacl_HKDF_stubs)
 module Hacl_NaCl = Hacl_NaCl_bindings.Bindings(Hacl_NaCl_stubs)
-module Hacl_Hash_Blake2 = Hacl_Hash_Blake2_bindings.Bindings(Hacl_Hash_Blake2_stubs)
-module Hacl_Blake2b_32 = Hacl_Hash_Blake2
-module Hacl_Blake2s_32 = Hacl_Hash_Blake2
+module Hacl_Blake2b_32 = Hacl_Hash_Blake2b_32_bindings.Bindings(Hacl_Hash_Blake2b_32_stubs)
```

Comment:
We should remove the the ocaml bindings here now. They diverged from what's published through hacl-packages and it will get painful the more we diverge.

@victor-dumitrescu @hacl-star/maintainers are we good to drop the ocaml bindings here or do we need anything else to happen before?

Reply:
Not sure how to proceed for the time being. I'm currently working on adding the OCaml bindings for https://github.com/hacl-star/hacl-star/blob/main/dist/gcc-compatible/Hacl_K256_ECDSA.h, which I don't think is in hacl-packages yet.

Other than for the opam files, have they diverged in the code itself?

Reply:
It's mostly the opam files I think. But it requires manually merging code, which is annoying and time consuming.

The plan is to finish up the CI integration, i.e. finish the remaining issues from #695 (comments with CI runs), and then do the move. This ensures that we get feedback on the ocaml bindings here in the hacl-star repo.